src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.WorkerThread.cs

```diff
@@ -316,25 +316,13 @@ private static bool TakeActiveRequest(PortableThreadPool threadPoolInstance)
 
             private static bool TryCreateWorkerThread()
             {
-                try
-                {
-                    // Thread pool threads must start in the default execution context without transferring the context, so
-                    // using UnsafeStart() instead of Start()
-                    Thread workerThread = new Thread(s_workerThreadStart);
-                    workerThread.IsThreadPoolThread = true;
-                    workerThread.IsBackground = true;
-                    // thread name will be set in thread proc
-                    workerThread.UnsafeStart();
-                }
-                catch (ThreadStartException)
-                {
-                    return false;
-                }
-                catch (OutOfMemoryException)
-                {
-                    return false;
-                }
-
+                // Thread pool threads must start in the default execution context without transferring the context, so
+                // using UnsafeStart() instead of Start()
+                Thread workerThread = new Thread(s_workerThreadStart);
+                workerThread.IsThreadPoolThread = true;
+                workerThread.IsBackground = true;
+                // thread name will be set in thread proc
+                workerThread.UnsafeStart();
                 return true;
```

Comment:
This always return true now, so you can also change the return type to `void` and delete check of the return value from the caller.